lib/livebook/runtime.ex

```diff
@@ -61,7 +61,7 @@ defprotocol Livebook.Runtime do
           # A control element
           | {:control, attrs :: map()}
           # Internal output format for errors
-          | {:error, message :: binary()}
+          | {:error, message :: binary(), type :: :unavailable_secret | :other}
```

Comment:
Currently the runtime is not aware of secrets. To keep that, we can use a bit more generic error:

```suggestion
          | {:error, message :: String.t(), type :: {:unavailable_env, String.t()} | :other}
```

And then on livebook side we match if the env var has `LB_` prefix. There is `exception.env`, @josevalim it should be fine to use that, right?